common/src/main/scala/uk/ac/wellcome/models/transformable/Transformable.scala

```diff
@@ -6,52 +6,46 @@ import uk.ac.wellcome.models.transformable.sierra.{
 }
 import io.circe.Decoder
 import cats.syntax.functor._
+import uk.ac.wellcome.models.Versioned
 import uk.ac.wellcome.utils.JsonUtil._
 
-sealed trait Transformable
-object Transformable {
```

Comment:
Don't need this IF and only if we ensure that the bib and item merger serialize stuff in s3 with the type discriminator. Should be as simple as using`versionedHybridStore.put(sierraTransformable: Transformable)` instead of `versionedHybridStore.put(sierraTransformable)`